thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/template.hbs

```diff
@@ -228,17 +230,19 @@
                   <li class="te-anomaly-table__list--lighter">{{anomaly.durationStr}}</li>
                 </ul>
                </td>
-               <td class="te-anomaly-table__cell">
-                <ul class="te-anomaly-table__list">
-                 {{#each anomaly.dimensionList as |dimension|}}
-                    <li class="te-anomaly-table__list--smaller">
-                      {{dimension.dimensionKey}}: <span class="stronger">{{dimension.dimensionVal}}</span>
-                    </li>
-                 {{else}}
-                    -
-                 {{/each}}
-                </ul>
-               </td>
+               {{#if alertHasDimensions}}
+                 <td class="te-anomaly-table__cell">
+                  <ul class="te-anomaly-table__list">
+                   {{#each anomaly.dimensionList as |dimension|}}
+                      <li class="te-anomaly-table__list--smaller">
+                        {{dimension.dimensionKey}}: <span class="stronger">{{dimension.dimensionVal}}</span>
+                      </li>
+                   {{else}}
+                      -
+                   {{/each}}
+                  </ul>
+                 </td>
+               {{/if}}
                <td class="te-anomaly-table__cell">{{anomaly.severityScore}}</td>
```

Comment:
Any reason why we can't use `ember-models-table` for this? I think I might have asked this in an earlier PR a while back, but I don't remember. If it doesn't make sense, be sure to include it in the comments. We should try to avoid building more components that we'll have to manage overtime instead of leveraging existing ones.

Reply:
So this table, if we choose to "componentize" it, can be refactored into ember-models-table with some time commitment due to custom filtering. We did not think it was wise to sink time into it in the MVP.